manifests/plugin.pp

```diff
@@ -27,7 +27,7 @@
   String[1] $config_file_owner = 'root',
   String[1] $config_file_group = $foreman::group,
   Stdlib::Filemode $config_file_mode = '0640',
-  Optional[String] $config = undef,
+  Optional[Variant[String, Sensitive[String]]] $config = undef,
```

Comment:
Should the `config_file` block below be changed to add `show_diff => !($config =~ Sensitive)` so it doesn't show up in logs anymore?

Reply:
`file`-Resource does not show sensitive Content.  No Need for `show_diff => false`.